core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -132,19 +184,23 @@ public void add(Supplier<JournalContext> journalContext, AlluxioURI alluxioUri,
         }
       }
 
-      Map<String, String> properties = options.getPropertiesMap();
-      mState.applyAndJournal(journalContext, AddMountPointEntry.newBuilder()
-          .addAllProperties(properties.entrySet().stream()
-              .map(entry -> StringPairEntry.newBuilder()
-                  .setKey(entry.getKey()).setValue(entry.getValue()).build())
-              .collect(Collectors.toList()))
-          .setAlluxioPath(alluxioPath)
-          .setMountId(mountId)
-          .setReadOnly(options.getReadOnly())
-          .setShared(options.getShared())
-          .setUfsPath(ufsUri.toString())
-          .build());
+      // Check that the alluxioPath we're creating doesn't shadow a path in the parent UFS
```

Comment:
Can you give an example of this case? And if an example can be given, pls add a unit test on that.